shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.h

```diff
@@ -30,8 +29,8 @@ typedef NS_ENUM(NSInteger, FlutterScribbleInteractionStatus) {
 @interface FlutterTextInputPlugin
     : NSObject <FlutterKeySecondaryResponder, UIIndirectScribbleInteractionDelegate>
 
-@property(nonatomic, assign) UIViewController* viewController;
-@property(nonatomic, assign) id<FlutterIndirectScribbleDelegate> indirectScribbleDelegate;
+@property(nonatomic, weak) UIViewController* viewController;
```

Comment:
Does flutter view controller retains the `FlutterTextInputPlugin` (opposite direction)?

Reply:
Yes `FlutterViewController` seems to retain an engine instance: `fml::scoped_nsobject<FlutterEngine> _engine`